demisto_sdk/commands/validate/tests/GF_validators_test.py

```diff
@@ -51,6 +54,26 @@ def test_GenericFieldGroupValidator_is_valid():
     assert not GenericFieldGroupValidator().is_valid([generic_field])
 
 
+def test_UnsearchableKeyValidator_is_valid():
+    """
+    Given:
+        - GenericField content items
+    When:
+        - run is_valid method
+    Then:
+        - Ensure that the ValidationResult returned
+          for the GenericField whose 'unsearchable' field is set to false or not or undefined
```

Comment:
`or undefined` is a great case to add to this test (current test is calling `.get` behind the scenes and `False`/missing behave the same, but it would be best to test explicitly)